### PR TITLE
Fix hermes not working when 'log' folder didn't exist

### PR DESCRIPTION
**Pull request by ThomasVy**
Issue was that Hermes was not working when the 'log' folder didn't exist. I fixed
the issue by checking if the folder existed and if it doesn't, the program will
create the directory.

### CODE REVIEW

**Comment by bill-luu** [comment]
Looks like the branch isn't synced up with master, and Travis is complaining about the formatting. Run the format.sh script to fix that issue

File: src/Tests/target_wrapper.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```
Comment (bill-luu) [inline src/Tests/target_wrapper.sh]: Was this file intended to be committed?

You can add these commands to your `.profile`, and then it'll be a part of your environment.
http://www.theunixschool.com/2011/07/what-is-profile-file.html?m=1
https://unix.stackexchange.com/questions/40708/what-is-the-difference-between-profile-bashrc-bash-profile-gnomer

**Review by bill-luu — DISMISSED** [review]

**Review by mglyngberg — APPROVED** [review]
